src/components/LeftDrawerOrg/LeftDrawerOrg.tsx

```
@@ -136,6 +150,45 @@ const leftDrawerOrg = ({
+        to={item.path.replace(':orgId', orgId)}
+        key={item.pluginId}
+        onClick={handleLinkClick}
+      >
+        {({ isActive }) => (
+          <button
+            className={
+              isActive
+                ? styles.leftDrawerActiveButton
+                : styles.leftDrawerInactiveButton
+            }
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)